lib/Router.js

```diff
@@ -371,12 +371,12 @@ class Router extends EnhancedEventEmitter_1.EnhancedEventEmitter {
      * @async
      * @returns {ShmTransport}
      */
-    async createShmTransport({ listenIp, shm, log, appData = {} }) {
```

Comment:
Please leave appData an object for consistency with other similar transport creating APIs. We assume that it is an object and it may contain information about callId, peerId, streamName and mirrorId. I am testing the branch where I actually pick that data in shm and add into logs.
If you are passing a special string for shm apps it would be better to add it under shm object (the 2nd param in this function).

Reply:
thanks, will do